public/icons/chevron-down.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M19.6607 32.6271L20 32.9404L20.3393 32.6271L34.4138 19.627C35.5777 18.5518 37.4801 18.5518 38.6441 19.627C39.7853 20.6811 39.7853 22.3724 38.6441 23.4265L22.1152 38.6937C20.9512 39.7688 19.0488 39.7688 17.8848 38.6937L1.35591 23.4265C0.214696 22.3724 0.214696 20.6811 1.35591 19.627C2.51989 18.5518 4.42226 18.5518 5.58624 19.627L19.6607 32.6271ZM19.6607 14.3065L20 14.6198L20.3393 14.3065L34.4138 1.30634C35.5777 0.23122 37.4801 0.231219 38.6441 1.30634C39.7853 2.36044 39.7853 4.05177 38.6441 5.10587L22.1152 20.373C20.9512 21.4482 19.0488 21.4482 17.8848 20.373L1.35591 5.10587C0.214697 4.05177 0.214696 2.36044 1.35591 1.30634C2.51989 0.23122 4.42226 0.23122 5.58624 1.30634L19.6607 14.3065Z" fill="white" stroke="#080D26"/>
+</svg>
```

Comment:
Take a look at the [MUI icons](https://mui.com/components/material-icons/), you probably don't need to add a custom SVG. In particular `KeyboardDoubleArrowDown` and `ExpandMore` might be what you want.